requirements/base.txt

```diff
@@ -2,7 +2,7 @@ chevron~=0.12
 click~=8.0
 #2.1.x and later versions of Werkzeug library has a bug for setting up <path:*> endpoints
 Flask<2.1
-Werkzeug<2.1
+Werkzeug<2.3
```

Comment:
Is this still a concern? Seems like the tests are passing (ignoring the one suite that failed to initialize)

Reply:
Running local integration tests appear to all be passing on my local system, looks like this is fine